### ISSUE TITLE
MissingBlankLineAfterImports false positive with wildcard import as last import

### ISSUE THREAD

**Issue by treyturner** [CodeNarc/CodeNarc#546]
Hi, I'm linting Groovy code with CodeNarc v1.6 via the Gradle plugin.

I have the `NoWildcardImports` rule disabled, and have found that if a wildcard import is the last import, then I get false positives for `MissingBlankLineAfterImports`. If I add a second blank line after the imports, I get a `ConsecutiveBlankLines` error. For workarounds, I can either rearrange my imports so that the wildcard import isn't last, or I can avoid use of wildcard imports. It doesn't seem to happen if my only import is a wildcard.

This seems like a CodeNarc bug, as I find wildcard imports acceptable for my purposes.

I additionally found this false positive is thrown if the last import is commented out.

Example where wildcard import is last:
```groovy
package com.example

import groovy.util.logging.Slf4j
import spock.lang.*

@Slf4j
class ExampleSpec extends Specification {
    // ...
}
```

Example where last import is commented out:
```groovy
package com.example

import spock.lang.Specification
//import spock.lang.Ignore

class ExampleSpec extends Specification {
    // ...
}
```

Thanks for your time and work on CodeNarc.

**Comment by treyturner** [comment]
Just discovered this is a duplicate of #517 / #518 , at least in the case of the wildcard imports. Not sure about a commented import being last yet

**Comment by treyturner** [comment]
Closing this because the main issue is already fixed, and commented imports should be removed anyway. Thanks for the diligence, looking forward to this fix and #507
